katello/katello.spec

```diff
@@ -64,6 +64,8 @@ install -Dp -m0755 %{SOURCE4} %{buildroot}%{_bindir}/katello-service
 install -Dp -m0755 %{SOURCE5} %{buildroot}%{_sbindir}/service-wait
 install -Dp -m0755 %{SOURCE2} %{buildroot}%{_bindir}/katello-remove
 install -Dp -m0755 %{SOURCE1} %{buildroot}/usr/share/foreman/script/foreman-debug.d/katello-debug.sh
+install -Dp -m0755 script/katello-backup %{buildroot}%{_bindir}/katello-backup
+install -Dp -m0755 script/katello-restore %{buildroot}%{_bindir}/katello-restore
 
```

Comment:
you need a SOURCE6 and SOURCE7 like the other source files in this specfile, so it would be:

install -Dp -m0755 %{SOURCE7} %{buildroot}%{_bindir}/katello-backup
install -Dp -m0755 %{SOURCE6} %{buildroot}%{_bindir}/katello-restore

just be sure to define SOURCE6 and 7 above with the rest